sympy/physics/mechanics/kane.py

```diff
@@ -244,7 +255,8 @@ def _initialize_kindiffeq_matrices(self, kdeqs):
             Kinematic differential equations in the form of f(u,q',q,t) where
             f() = 0. The equations have to be linear in the generalized
             coordinates and generalized speeds.
-
+        explicit_kinematics: bool
+            If True, force the kinematic equations to be in explicit form
```

Comment:
The qdots are still solved for, so it isn't clear how this forces things to an explicit form.